types/hapi/index.d.ts

```diff
@@ -440,7 +440,7 @@ export interface Request extends Podium {
      * The request payload based on the route payload.output and payload.parse settings.
      * TODO check this typing and add references / links.
      */
-    readonly payload: stream.Readable | Buffer | string | object | null;
+    readonly payload: stream.Readable | Buffer | string | object;
```

Comment:
Technically this is valid, the `payload` can and will often be null.

Reply:
I didn't have enough context to be more surgical in reverting the original change.